sensor/utils/utils.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"k8s.io/client-go/kubernetes"
+)
+
+// HasAPI checks whether the kubernetes client supports the gv API for the specified kind
+func HasAPI(client kubernetes.Interface, gv, kind string) (bool, error) {
```

Comment:
Let's please rename `gv` to `groupVersion` because `kind` is not shortened to `k`.
```suggestion
// HasAPI checks whether the kubernetes client supports the groupVersion API for the specified kind
func HasAPI(client kubernetes.Interface, groupVersion, kind string) (bool, error) {
```